doc/devel/contributing.rst

```diff
@@ -435,18 +435,22 @@ To include `logging` in your module, at the top of the module, you need to
 will log to a logger named ``matplotlib.yourmodulename``.
 
 If an end-user of Matplotlib sets up `logging` to display at levels
-more verbose than `logger.WARNING` in their code as follows::
+more verbose than `logger.WARNING` in their code with the Matplotlib-provided
+helper::
+
+  matplotlib.set_loglevel("debug")
```

Comment:
I think this should be `plt.set_loglevel`.  Most folks import pyplot, they don't always import matplotlib....

Reply:
done